lms/urls.py

```diff
@@ -548,11 +557,11 @@
         name='debug_cohort_mgmt',
     ),
     url(
-        r'^courses/{}/cohorts/topics$'.format(
+        r'^courses/{}/discussion/topics$'.format(
             settings.COURSE_KEY_PATTERN,
         ),
-        'openedx.core.djangoapps.course_groups.views.cohort_discussion_topics',
-        name='cohort_discussion_topics',
+        'openedx.core.djangoapps.course_groups.views.divided_discussion_topics',
+        name='divided_discussion_topics',
```

Comment:
What does this return? Just the divided discussions, or a list of all discussion with information about which are divided?

If it contains the list of all discussions, I would call it "discussion_topics", because as-is, it seems like it only returns the discussion topics that are divided (and perhaps that's true... I don't know).